packages/fxa-auth-server/lib/routes/subscriptions/paypal.ts

```diff
@@ -17,7 +17,9 @@ import Container from 'typedi';
 
 import { ConfigType } from '../../../config';
 import error from '../../error';
+import { COUNTRIES_LONG_NAME_TO_SHORT_NAME_MAP } from '../../payments/customer-locations';
```

Comment:
...I guess customer-locations isn't used _only_ in the one time script now.  😢

Reply:
IIUC, this will load the CustomerLocations module now and anything else it loads as part of the fxa-auth-server. Is the concern about just unnecessarily loading this module when I just need a static file? E.g. unnecessary loading. Or are there risks around dependencies (though I don't use `Container` here)?

Reply:
Either way, it's a good point (though I just noticed we also import this object from `customer-locations.ts` in lib/routes/subscriptions/stripe.ts). In the interest of time for today's dot release, since it's not blocking, I will handle moving this in FXA-4224 (#11070 ).

Reply:
No real concerns.  Mainly just want keep code for one time scripts separate from services code so it's easier to clean up when deleting the one time code.